### PR TITLE
PMM-12776 Re-enable RBAC

### PR DESCRIPTION
**Pull request by BupycHuk**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: pkg/services/pluginsintegration/clientmiddleware/forwarded_percona_token_middleware.go [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+	if pReq == nil {
+		return ctx
+	}
+
+	mw := httpclient.NamedMiddlewareFunc(forwardedProxyFilterMiddlewareName, func(opts httpclient.Options, next http.RoundTripper) http.RoundTripper {
+		return httpclient.RoundTripperFunc(func(req *http.Request) (*http.Response, error) {
+			reqCtx := contexthandler.FromContext(ctx)
+
+			// @PERCONA
+			// @PERCONA_TODO
```
Comment (JiriCtvrtka) [inline pkg/services/pluginsintegration/clientmiddleware/forwarded_percona_token_middleware.go:40]: Is there still todo?

**Review by JiriCtvrtka — APPROVED** [review]

File: Makefile [hunk 2]
```diff
@@ -336,3 +336,11 @@ format-drone:
 
 help: ## Display this help.
 	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_-]+:.*?##/ { printf "  \033[36m%-15s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
+
+rebuild-pmm:
+		make build-go
+		rm -f /usr/sbin/grafana-server
+		rm -f /usr/sbin/grafana
+		cp ./bin/linux-amd64/grafana-server /usr/sbin/grafana-server
```
Comment (ademidoff) [inline Makefile:344]: Both `grafana-server` and `grafana-cli` are not used, so we can remove this line.

File: Makefile [hunk 3]
```diff
@@ -336,3 +336,11 @@ format-drone:
 
 help: ## Display this help.
 	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_-]+:.*?##/ { printf "  \033[36m%-15s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
+
+rebuild-pmm:
+		make build-go
+		rm -f /usr/sbin/grafana-server
```
Comment (ademidoff) [inline Makefile:342]:
```suggestion
		rm -f /usr/sbin/grafana-server /usr/sbin/grafana-cli
```

File: pkg/services/pluginsintegration/pluginsintegration.go [hunk 4]
```diff
@@ -189,6 +189,8 @@ func CreateMiddlewares(cfg *setting.Cfg, oAuthTokenService oauthtoken.OAuthToken
 
 	middlewares = append(middlewares, clientmiddleware.NewHTTPClientMiddleware())
 
+	middlewares = append(middlewares, clientmiddleware.NewPerconaForwarderHTTPClientMiddleware())
```
Comment (matejkubinec) [inline pkg/services/pluginsintegration/pluginsintegration.go:192]: Please add a `@PERCONA` comment here, helps with seeing PMM modifications during grafana upgrade.

File: pkg/services/pluginsintegration/clientmiddleware/forwarded_percona_token_middleware.go (hunk 1 above)
Comment (BupycHuk) [reply pkg/services/pluginsintegration/clientmiddleware/forwarded_percona_token_middleware.go]: removed and moved

File: Makefile (hunk 2 above)
Comment (BupycHuk) [reply Makefile]: removed all mentions of grafana-server and grafana-cli

File: pkg/services/pluginsintegration/pluginsintegration.go (hunk 4 above)
Comment (BupycHuk) [reply pkg/services/pluginsintegration/pluginsintegration.go:193]: Added here and to the other file

**Review by matejkubinec — APPROVED** [review]

**Pull request merged**